packages/vite/src/executors/test/schema.d.ts

```diff
@@ -8,5 +8,5 @@ export interface VitestExecutorOptions {
   update?: boolean;
   reportsDirectory?: string;
   coverage?: boolean;
-  testFile?: string;
+  testFile?: string[];
```

Comment:
now that this is an array does it make sense to rename this to be plural? `testFiles` 🤔

Reply:
I'm not sure. I think this makes sense when using the option in `project.json` e.g.
```json
{
   "testFiles": ["tests/foo.test.tsx", "tests/bar.test.tsx"]
}
```
But I prefer `testFile` when using the CLI
```
nx run app:test --testFIle=foo.test.tsx --testFile=bar.test.tsx
```
over 
```
nx run app:test --testFIles=foo.test.tsx --testFiles=bar.test.tsx
```

It just personal preference, so happy to go with `testFiles` but just making the case for keeping as is

Reply:
Great, thanks for the suggestion, I've updated to use `testFiles` with a `testFile` alias